src/redux/actions/liquorSearch.js

```diff
@@ -0,0 +1,20 @@
+export const loadDrinkData = (data) => {
+    return {
+        type: 'LOAD_DRINK_DATA',
+        payload: data
+    }
+}
+
+export const setLiquorType = (drink) => {
```

Comment:
Quick thought on naming ... should we rename this parameter to 'liquorType' or 'type' instead of 'drink'?